test/functional/feature_fee_estimation.py

```diff
@@ -4,7 +4,7 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test fee estimation code."""
```

Comment:
nit:
this commit message and PR title are a bit long see https://chris.beams.io/posts/git-commit/ that was recommended in https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#committing-patches